src/Avalonia.Themes.Fluent/Controls/HeaderedContentControl.xaml

```diff
@@ -0,0 +1,42 @@
+<ResourceDictionary xmlns="https://github.com/avaloniaui"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    x:ClassModifier="internal">
+  <ControlTheme x:Key="{x:Type HeaderedContentControl}"
+                TargetType="HeaderedContentControl">
+
+    <Setter Property="Padding" Value="3" />
+    <Setter Property="Template">
+      <ControlTemplate>
+        <Border Name="PART_Border"
+                BorderBrush="{TemplateBinding BorderBrush}"
+                BorderThickness="{TemplateBinding BorderThickness}"
+                CornerRadius="{TemplateBinding CornerRadius}"
+                Background="{TemplateBinding Background}"
+                Padding="{TemplateBinding Padding}">
+          <StackPanel Orientation="Vertical">
```

Comment:
Wouldn't it make more sense if it was a DockPanel? Or Grid?
Where PART_HeaderPresenter has DockPanel.Dock="Top"

Reply:
This way content won't have infinite height by default, making it usable with list controls